frontend/packages/dev-console/src/components/gitops/GitOpsListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Grid, GridItem, Card, CardBody } from '@patternfly/react-core';
+import { ResourceIcon, history } from '@console/internal/components/utils';
+import { GitOpsAppGroupData } from './gitops-types';
+import './GitOpsList.scss';
```

Comment:
Should have a separate scss file corresponding to this component.
```suggestion
import './GitOpsListItem.scss';
```